EventListener/ValidationRequestListener.php

```diff
@@ -88,8 +91,9 @@ public function onKernelRequest(GetResponseEvent $event)
             $dataObject = new \stdClass();
         }
 
-        // Validate data with given schema.
-        $validator = new \JsonSchema\Validator();
+        $schemaStorage = new SchemaStorage();
+        $schemaStorage->addSchema('file://' . $schemaFile, $schema);
```

Comment:
Yes definitely. It took a quick look at it and as far as I have seen there is no caching mechanism implemented within the SchemaStorage. We would have to do this our own. I thought of adding some caching mechanism of Symfony in another PR.